db-changes/studies/28102021_alter_study_user.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE study_user RENAME COLUMN receive_anonymization_report TO receive_study_user_report;
```

Comment:
Also, I think this needs to be done on import too.

Reply:
done, added to import too